public/app/core/components/AccessControl/AddPermission.tsx

```diff
@@ -72,6 +78,10 @@ export const AddPermission = ({ title = 'Add Permission For', permissions, assig
               <UserPicker onSelected={(u) => setUserId(u.value || 0)} className={'width-20'} />
             )}
 
+            {target === PermissionTarget.ServiceAccount && (
```

Comment:
We add service accounts as an option for resource permission, but actually, backend `/api/access-control/<service>/description` does not return `ServiceAccounts` as an assignments for any services (ie dashboards and folders) except team permissions (see `pkg/services/accesscontrol/ossaccesscontrol/permissions_services.go`). I'm not 100% sure if it by intention or not, could you clarify?

Reply:
I think it doesn't make much difference if the service account option is simply skipped or set to `false`. But I'll set it to `false` to make it explicit that we currently don't support dash/folder resource permissions for service accounts.

One thing to note is that dash/folder resource permissions can actually be assigned to service accounts, as they use the same backend code as user assignments. It just isn't supported through the UI.

Reply:
Ok, got it!